go/vt/vtorc/logic/tablet_discovery.go

```diff
@@ -69,36 +67,31 @@ func OpenTabletDiscovery() <-chan time.Time {
 	if _, err := db.ExecVTOrc("delete from vitess_tablet"); err != nil {
 		log.Error(err)
 	}
-	// We refresh all information from the topo once before we start the ticks to do it on a timer.
-	populateAllInformation()
+	// We refresh all information from the topo once before we start the ticks to do
+	// it on a timer. We can wait forever (context.Background()) for this call.
```

Comment:
Can you please explain how do you mean "We can wait forever (context.Background()) for this call."?

Reply:
@shlomi-noach this comment is confusing, yeah

What I intended with adding the `context.Context` is to prevent [ticks from overlapping here](https://github.com/vitessio/vitess/pull/17129/files#diff-a0f5bcadcb7ba9b23769c7462b1a694210928f39bb8002105c404c4d6ea56f06R333)

This area of the code is ran once before the ticker starts at vtorc init time, so I figured waiting forever had no risk. Adding a timeout here as well wouldn't hurt either I suppose

Reply:
> is to prevent ticks from overlapping here

Sorry, not "overlapping" per se, but one loop iteration stalling things

Reply:
I do prefer that we have some form of timeout -- let's say this errors and we cannot connect to topo -- is there going to be some retry? If yes, let's timeout. If not, let's consider our options.

Reply:
@shlomi-noach makes sense, I've updated to use a timeout == `topo.RemoteOperationTimeout` 👍